package.json

```diff
@@ -38,8 +38,9 @@
     "clean": "npm run clean:typescript && rimraf node_modules && npm cache clean",
     "clean:typescript": "rimraf dist",
     "lint": "tslint lib/**/*.ts",
-    "rollup": "./node_modules/.bin/rollup -c rollup.config.js dist/angular-tree-component.js -o dist/angular-tree-component.umd.js",
-    "build": "npm run lint && npm run clean:typescript && ./node_modules/.bin/ngc && npm run rollup && cp lib/angular-tree-component.css dist",
+    "rollup": "./node_modules/.bin/rollup -c rollup.config.js dist/lib/angular-tree-component.js -o dist/angular-tree-component.umd.js",
+    "build": "npm run lint && npm run clean:typescript && npm run build:ng9 && npm run rollup && cp lib/angular-tree-component.css dist",
+    "build:ng9": "./node_modules/.bin/ngcc && ./node_modules/.bin/ngc",
```

Comment:
This should also be part of example:cli and example:cli:win scripts (instead of running just ngc), because now they don't work

Reply:
I guess I wasn't seeing that error since I had already ran a build; I added the `build:ng9` script to  `example:cli`, and `build:win:ng9` to `example:cli:win`. Do I need to run `example:cli:win` somewhere in particular to confirm it works? Right now the script doesn't work in master or my branch.

Reply:
I haven't tested it either, because I don't have windows